### DISCUSSION TITLE
going back after sending a message should return to main page instead of contacts list

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#493] [Ideas]
# steps to reproduce

1. From main page, press +
2. Choose a contact
3. Write a message
4. Send it
5. Press back button

# actual behavior
Back button brings you to the contacts list

# desired behavior
Since the user already sent a message to the contact, back button should bring to the main page.

As far as I know, no other conversation apps has this behavior

**Comment by Aga-C** [comment]
Please create a bug report for that.

**Comment by ghost** [comment]
Ok, but it is not a bug...

**Reply by Aga-C** [reply]
We will tag it as an enhancement. You can raise it how you prefer, but Issues is a better place for it.
